### PR TITLE
Update README to add section RE value of dapps (nontechnical), add links to other docs, section reorganization

### PR DESCRIPTION
**Pull request by EricLWang**
- restructured sectioning and adding concise description of value add for Tasit mobile apps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,31 @@
...
 
-### Introduction
+## What is Tasit?
 
-Tasit is a suite of mobile apps for mainstream users that lets them use popular Ethereum dapps.
+Tasit is 2 things: 
 
-Unlike with Status, Coinbase Wallet, Cipher, or other dapp browsers, you can use the dapp via a standalone native mobile app.
+- 1 ) a suite of native mobile apps for mainstream users that lets them use popular Ethereum dapps in the same manner as their other favorite apps (e.g. Instagram, Facebook, Gmail, etc.) 
+- 2 ) a [Javascript SDK](https://github.com/tasitlabs/TasitSDK) that empowers the creation of even more native Ethereum dapps.
```
Comment (pcowgill) [inline README.md]: I think you can empower someone with something, but I don't think you can empower creation itself

File: README.md [hunk 2]
```diff
@@ -4,18 +4,31 @@
...
+_Disclaimer:_ The alpha versions of these apps are under active development. We encourage you to try them out today for hackathons, etc., but they're not ready for anything involving real funds on mainnet. If you'd prefer to wait for a more battle-tested release, please watch this repo with the `Releases only` setting and/or sign up to be notified about our releases on the [tasit.io](https://tasit.io) website.
 
 The apps are all built using the [Tasit SDK](https://github.com/tasitlabs/tasitsdk) and React Native.
 
+## Why are native dapps important?
+
+One major bottleneck and obstacle in the mainstream adoption of dapps is the clunky and confusing UX of today's dapps - typically served within a mobile browser (more on that [here](https://github.com/tasitlabs/TasitSDK#why-from-the-users-perspective)). 
+
+The focus for the Tasit products is on providing simple, third-party clients for popular dapps. The Tasit apps will provide  delightful UX and simpler onboarding for the dapp and in some cases extra features too.
```
Comment (pcowgill) [inline README.md]:
```suggestion
The focus for the Tasit products is on providing simple, third-party clients for popular dapps. The Tasit apps will provide delightful UX and simpler onboarding for the dapp and in some cases extra features too. 
```

File: README.md [hunk 3]
```diff
@@ -4,18 +4,31 @@
...
 
+## Why are native dapps important?
+
+One major bottleneck and obstacle in the mainstream adoption of dapps is the clunky and confusing UX of today's dapps - typically served within a mobile browser (more on that [here](https://github.com/tasitlabs/TasitSDK#why-from-the-users-perspective)). 
+
+The focus for the Tasit products is on providing simple, third-party clients for popular dapps. The Tasit apps will provide  delightful UX and simpler onboarding for the dapp and in some cases extra features too. 
+
+From a developer's perspective, a lack of tooling means a great deal of grunt setup work is required with every project, slowing down product development and taking valuable time and focus way from business logic. (more on that [here](https://github.com/tasitlabs/TasitSDK#why-from-a-developers--ethereum-enthusiasts-perspective)).
+
+With the two pronged approach of the Tasit native apps and the Tasit SDK, we hope to achieve the aim of mainstream Ethereum dapp adoption by making the experience significantly smoother and more enjoyable for both users and builders.
```
Comment (pcowgill) [inline README.md]:
```suggestion
Through the two-pronged approach with the Tasit native apps and the Tasit SDK, we hope to achieve the aim of mainstream Ethereum dapp adoption by making the experience significantly smoother and more enjoyable for both users and builders. 
```

File: README.md [hunk 4]
```diff
@@ -4,18 +4,31 @@
...
+
+One major bottleneck and obstacle in the mainstream adoption of dapps is the clunky and confusing UX of today's dapps - typically served within a mobile browser (more on that [here](https://github.com/tasitlabs/TasitSDK#why-from-the-users-perspective)). 
+
+The focus for the Tasit products is on providing simple, third-party clients for popular dapps. The Tasit apps will provide  delightful UX and simpler onboarding for the dapp and in some cases extra features too. 
+
+From a developer's perspective, a lack of tooling means a great deal of grunt setup work is required with every project, slowing down product development and taking valuable time and focus way from business logic. (more on that [here](https://github.com/tasitlabs/TasitSDK#why-from-a-developers--ethereum-enthusiasts-perspective)).
+
+With the two pronged approach of the Tasit native apps and the Tasit SDK, we hope to achieve the aim of mainstream Ethereum dapp adoption by making the experience significantly smoother and more enjoyable for both users and builders. 
+
+## Tasit Apps
```
Comment (pcowgill) [inline README.md]: Now that there's this header, we should probably move `Getting started` out of this section

File: README.md [hunk 5]
```diff
@@ -85,17 +98,17 @@ This app scaffold comes ready to submit for beta testing on iOS with Testflight
 
 ### Coming soon
 
-##### A Tasit list app
+##### Tasit list app
```
Comment (pcowgill) [inline README.md]: Do you think with the "a" gone it's still clear that we won't be making a generalist app that supports all list dapps, but building a mobile app for a specific list dapp?

I think it maybe still be clear - just checking. Thanks!

**Comment by pcowgill** [comment]
@EricLWang If you can rename the PR with a more descriptive name, that would also be a big help for when we're looking back through old PRs. Thanks!

File: README.md (hunk 4 above)
Comment (EricLWang) [reply README.md]: agreed - addressed

File: README.md (hunk 1 above)
Comment (EricLWang) [reply README.md]: updated to `empowers developers to create`

File: README.md (hunk 5 above)
Comment (EricLWang) [reply README.md]: agree that the `a` provides greater specificity while a dapp project is being determined

File: README.md (hunk 4 above)
Comment (pcowgill) [reply README.md]: Thanks! Resolving this.

**Pull request merged**